django/core/cache/backends/memcached.py

```diff
@@ -35,6 +35,14 @@ def _cache(self):
 
         return self._client
 
+    def _get(self, key, default=None):
```

Comment:
Rather than introducing a new method, I would rather remove `if val is None:  return default` from `BaseMemcachedCache.get()` and override `MemcachedCache.get()` with the workaround -- then that method can be removed when https://github.com/linsomniac/python-memcached/pull/158 is addressed.

Reply:
Good point, thannks